ui/imports/shared/views/DeviceSyncingView.qml

```diff
@@ -132,8 +133,10 @@ Item {
             visible: !d.pairingFailed
             subTitle: qsTr("Synced device")
             enabled: false
-            loading: d.pairingInProgress
-            deviceName: qsTr("No device name")
+//            loading: d.pairingInProgress
+//            loading: root.installationId === ""
```

Comment:
leftovers?

Reply:
Sorry, this is from a different issue 😄  
Removed this commit.